packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp

```diff
@@ -325,13 +325,19 @@ evalModelImpl(const Thyra::ModelEvaluatorBase::InArgs<Scalar> &inArgs,
     typedef typename tpetra_vec::execution_space execution_space;
 
     // Create ghosted objects
+    // Should there be braces corresponding to this if-statement and the following lines of code (as in xPtr_ case below)?
     if (is_null(uPtr_))
       uPtr_ = Teuchos::rcp(new tpetra_vec(xGhostedMap_));
+
+      if (uPtr_->need_sync_host())
+        uPtr_->sync_host();
       uPtr_->modify_host();
```

Comment:
@rppawlo Why are you calling `modify_host` on the target of an Import?  It looks like this was intended to force MultiVector to pack and unpack on host.  That doesn't necessarily work like you think it does any more.  For example, the target of an Export or Import has the right to control its own sync state.

Reply:
@mhoemmen - not sure. This was written by someone who has left sandia, so I can't get a quick answer. Can we just remove the modify calls?

Reply:
@rppawlo We should remove them, unless downstream code implicitly depends on MultiVector being sync'd to host.  There's no good way to test for that, though.

Reply:
> We should remove them, unless downstream code implicitly depends on MultiVector being sync'd to host

@rppawlo @mhoemmen I'll hold off making any changes until getting a definite "yes" to remove the `modify_host` calls (and the sync checking I added in this PR)

Reply:
yes - let's remove them.